tests/scripts/pull_update_push_image.sh

```diff
@@ -206,7 +206,7 @@ if [ ! -z "${username}" ]; then
 fi
 
 # Upload image to target registry
-skopeo copy --dest-tls-verify=false ${multiarch_arg} ${credentials_args} ${local_image_ref_skopeo} docker://${remote_dest_image_ref}
+skopeo --override-os="linux" --override-arch="amd64" copy  --dest-tls-verify=false ${multiarch_arg} ${credentials_args} ${local_image_ref_skopeo} docker://${remote_dest_image_ref}
```

Comment:
In case of multiarch, would all images for all platforms be copied to the registry?

Reply:
Same as above, the platform override only happens for single-arch images